docs/source/guides/speed.rst

```diff
@@ -76,8 +76,8 @@ Prefer DDP over DP
 Whereas :class:`~pytorch_lightning.plugins.training_type.DDPPlugin` only performs 1 transfer to sync gradients, making DDP MUCH faster than DP.
 
 
-When using DDP set find_unused_parameters=False
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+When using DDP or DDPSpawn, set find_unused_parameters=False
```

Comment:
the term DDP already covered the family of ddp plugins. spawn is not the only one, there are others. 
so I think a simple note that this applies to all plugins supporting the `find_unused_parameters` would suffice

Reply:
Updated!

Reply:
love it